integration-tests/features/multimodule_analysis.feature

```diff
@@ -1,52 +1,51 @@
-Feature: cpp-multimodule-project
-
-  Test multimodule project with reports at root of the project
-
-  Scenario: cpp-multimodule-project
-      GIVEN the project "cpp-multimodule-project"
-      WHEN I run "sonar-runner -X"
-      THEN the analysis finishes successfully
-          AND the analysis log contains no error/warning messages except those matching:
-              """
-              .*WARN.*Unable to get a valid mac address, will use a dummy address
-              .*WARN.*cannot find the sources for '#include <gtest/gtest\.h>'
-              .*WARN.*cannot find the sources for '#include <iostream>'
-              .*WARN  - Cannot find the file '.*', skipping violations
-              .*WARN.*to create a dependency with .*
-              .*WARN  - the include root '.*' doesn't exist
-              .*WARN  - .* cannot find the sources for .*
-              .*WARN  - SCM provider autodetection failed.*
-              .*WARN.*Cannot find a report for '.*'
-              """
-          AND the following metrics have following values:
-               | metric                   | value |
-               # size metrics
-               | ncloc                    | 56    |
-               | lines                    | 150   |
-               | statements               | 36    |
-               | classes                  | 1     |
-               | files                    | 8     |
-               | directories              | 4     |
-               | functions                | 5     |
-               # comments / documentation
-               | comment_lines_density    | 30    |
-               | comment_lines            | 24    |
-               # duplications
-               | duplicated_lines_density | 57.3  |
-               | duplicated_lines         | 86    |
-               | duplicated_blocks        | 2     |
-               | duplicated_files         | 2     |
-               # complexity
-               | complexity               | 7     |
-               | function_complexity      | 1.4   |
-               | file_complexity          | 0.9   |
-               | class_complexity         | 6     |
-               # violations
-               | violations               | 28    |
-               # test execution statistics
-               | test_success_density     | 33.3  |
-               | test_failures            | 2     |
-               | skipped_tests            | 1     |
-               | test_errors              | 0     |
-               | tests                    | 3     |
-
+# Feature: cpp-multimodule-project
```

Comment:
i am ensure about this, with this change we are telling people that they must to have their reports always per module and not once at the root of the repo as it has been supported until now. as far as i know this work is mostly manual and you need to generate reports per module instead of generating the reports once for the all repo.

although this might be ok for some, i think we should also support the lazy people that just put reports on root of the repository and let the plugin deal with it. as far as i know the only way of doing it is with the reactor, so i think we should not remove it